atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueCacheImplTest.java

```diff
@@ -83,7 +83,18 @@
     @Before
     public void before() {
         eventCache = LockWatchEventCacheImpl.create(MetricsManagers.createForTests());
-        valueCache = new LockWatchValueCacheImpl(eventCache);
+        valueCache = new LockWatchValueCacheImpl(eventCache, 20_000, 0.0);
+    }
+
+    @Test
+    public void valueCacheCreatesValidatingTransactionCaches() {
```

Comment:
This test is testing that it caches reads, not that it does validation

Reply:
IIRC this test confirms that we read from remote twice, whereas we wouldn't if this was not the validating cache.

Reply:
(it's a bit indirect, but this would fail if this is not a validating cache).

Reply:
Ah right